Content.Client/Arcade/BlockGameMenu.cs

```diff
@@ -160,7 +160,7 @@ public BlockGameMenu(BlockGameBoundUserInterface owner)
 
             _newGameButton = new Button
             {
-                Text = Loc.GetString("New Game"),
+                Text = Loc.GetString("blockgame-new-game"),
```

Comment:
Sorry, didn't mention this yesterday but, maybe make these `blockgame-button-new-game`, `blocgkame-button-scoreboard` etc.

Just in general try to include a *little* bit more context